suites/pacific/rados/tier_3_rados.yaml

```diff
@@ -0,0 +1,50 @@
+tests:
+
+  # Cluster deployment stage
+
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                registry-url: registry.redhat.io
```

Comment:
Let's avoid this key/value... it breaks execution in IBM-C

Reply:
Fixed the issue.